src/code42cli/cmds/high_risk_employee.py

```diff
@@ -16,8 +18,18 @@
 from code42cli.errors import Code42CLIError
 from code42cli.file_readers import read_csv_arg
 from code42cli.file_readers import read_flat_file_arg
+from code42cli.options import format_option
 from code42cli.options import sdk_options
 
+
+filter_option = click.option(
+    "--filter",
+    help="High risk employee filter options. Defaults to OPEN.",
```

Comment:
Naming the default `OPEN` here is a bit confusing, as I was kind of assuming you'd be able to alternately filter by `CLOSED`, which doesn't seem to be an option. 

Could we just expose `EXFILTRATION_24_HOURS|EXFILTRATION_30_DAYS|LEAVING_TODAY` as available filter options and say the default is "no filter"?

Reply:
How about `ALL` ?